packages/x-data-grid-generator/tsconfig.json

```diff
@@ -4,11 +4,11 @@
     "types": [
       "@mui/internal-test-utils/initMatchers",
       "@mui/material/themeCssVarsAugmentation",
-      "chai-dom",
-      "mocha",
+      "vitest/globals",
       "node",
       "react"
-    ]
+    ],
+    "skipLibCheck": true
```

Comment:
Had to add this to every package due to the following issue:
<img width="1217" alt="Screenshot 2025-06-17 at 17 06 38" src="https://github.com/user-attachments/assets/28fa1489-7546-437e-8bc7-7551ab168a4c" />
It stops type-checking node_modules and "hides" this issue.

https://github.com/mui/material-ui/blob/6c0f14b50dc7c86134b8bb549da47dc33bf8b06a/packages-internal/test-utils/src/chaiPlugin.ts#L3
We could revert these once Core is migrated to vitest. 🤔